src/Nodes.ts

```diff
@@ -969,21 +970,35 @@ export class QueryVectorStoreOutResultsItem extends Future<
 
 /** ErrorOut */
 export class ErrorOut extends Future<Object> {
+  // prop = type
+  // prop.type = string
+  // prop.ref = None
+  // prop_return_class = Future<string>
+  // use_proxy = False
```

Comment:
I'm planning to remove this information form the codegen output, but I've been using it to debug for now.